packages/embed/src/utils/setup-google-analytics-instance.ts

```diff
@@ -84,19 +83,10 @@ export const setupGaInstance = (iframe: HTMLIFrameElement, embedId: string, shar
       if (tracker) {
         sendGaIdMessage(embedId, tracker.get('clientId'), iframe)
       } else {
-        logError(
-          `Whoops! You enabled the shareGaInstance feature in your` +
-            `typeform embed but the tracker with ID ${trackingId} was not found. ` +
-            'Make sure to include Google Analytics Javascript code before the Typeform Embed Javascript' +
-            'code in your page and use correct tracker ID. '
-        )
+        logErrorMessage(`the tracker with ID ${trackingId} was not found`)
       }
     } catch (exception) {
-      logError(
-        'Whoops! You enabled the shareGaInstance feature in your typeform embed but the Google Analytics ' +
-          'object has not been found. Make sure to include Google Analytics Javascript code before the ' +
-          'Typeform Embed Javascript code in your page. '
-      )
+      logErrorMessage('the Google Analytics object has not been found')
```

Comment:
Just to keep consistent with the other messages.
```suggestion
      logErrorMessage('the Google Analytics object was not found')
```